salt/modules/kubernetes.py

```diff
@@ -0,0 +1,500 @@
+'''
+Module for handling kubernetes calls.
+
+:optdepends:    - kubernetes Python client
+:configuration: This module is not usable until the following are specified
+    either in a pillar or in the minion's config file::
+
+        kubernetes.user: admin
+        kubernetes.password: verybadpass
+        kubernetes.api_url: 'http://127.0.0.1:8080'
+'''
+# Import Python Futures
+from __future__ import absolute_import
+from six import iteritems
```

Comment:
We'd like to use the version of `six` that we bundle. This could be `import salt.ext.six as six`.